XPlanning/data/mobilerobot/missions/mission-weh.json

```diff
@@ -0,0 +1,26 @@
+{
+	"map-file": "WEH4-scaled.json",
+	"start-id": "l38",
+	"goal-id": "l35",
+	"preference-info": [
+		{
+			"scaling-const": 0.5, 
```

Comment:
Here, the preference scaling is a bit more understandable. We say the relative weights between time, collision, and intrusiveness are 0.5, 0.4, 0.1. Meaning, we don't care too much in this mission about intrusiveness, and we care slightly more about time than collision.